server/paymentProviders/stripe/virtual-cards.js

```diff
@@ -121,6 +119,24 @@ const setCardStatus = async (virtualCard, status = 'canceled' | 'active' | 'inac
   return data;
 };
 
+const recordInActivityLog = async (virtualCard, amount, currency) => {
```

Comment:
Not sure if necessary to abstract this function if it is being used in a single place.